src/sentry/models/release.py

```diff
@@ -30,6 +30,7 @@ class ReleaseProject(Model):
 
     project = FlexibleForeignKey('sentry.Project')
     release = FlexibleForeignKey('sentry.Release')
+    new_groups = BoundedPositiveIntegerField(default=0, null=True)
```

Comment:
Ok, so we're going to have to drop this `default` in light of what South is generating for the migration.